src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Pages/Authentication.razor

```diff
@@ -0,0 +1,39 @@
+@page "/authentication/{action}"
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+@using Microsoft.Extensions.Options
+
+@*#if (IndividualLocalAuth)
+@*#if (Hosted)
+@inject IOptions<RemoteAuthenticationOptions<ApiAuthorizationProviderOptions>> Options
+#else
+@inject IOptions<RemoteAuthenticationOptions<OidcProviderOptions>> Options
+#endif*@
+#endif*@
+@*#if (IndividualB2CAuth || OrganizationalAuth)
```

Comment:
Does the templating language support an "else if" construct? If so it would be preferable to have that here, showing that we can't have two different things both injected called `Options`.

If it doesn't support that, please ignore this comment!